api/config/config_local.json

```diff
@@ -3,7 +3,7 @@
   "firecloud": {
     "baseUrl": "https:\/\/firecloud-orchestration.dsde-dev.broadinstitute.org",
     "billingAccountId": "014D91-FCB792-33D2C0",
-    "billingProjectPrefix": "aou-rw-local1-",
+    "billingProjectPrefix": "aou-rw-local2-",
```

Comment:
Revert?

Reply:
I actually ended up with a broken local... so I wanted to change it because this fixes it. (I think in the flurry of billing project fixes and breakages, my local got messed up somehow. Happy to revert back but I want to log that local2 has been used if we do, so there's not weird errors if we need to change it in the future.